src/scripts/wildBattle/RouteHelper.ts

```diff
@@ -38,6 +38,38 @@ class RouteHelper {
         return pokemonList;
     }
 
+    public static routePokerusEVs(route:number, region:GameConstants.Region): string {
+        const possiblePokemon: PokemonNameType[] = RouteHelper.getAvailablePokemonList(route, region);
+        if (this.minPokerus(possiblePokemon) == 3) {
+            return 'All Pokémon on this route are resistant!';
+        }
+        const currentEVs = this.getEvs(possiblePokemon);
+        return 'EVs until all Pokémon are resistant on this route: ${currentEVs} / ${50 * possiblePokemon.length}.';
```

Comment:
I think the automatic eslint might have ruined the strings. It needs to be ` and not ', as far as i remember

Reply:
Done :)